website/content/docs/enterprise/index.mdx

```diff
@@ -94,6 +116,14 @@ available for the specified Nomad task at hand.
 
 See the [Vault Integration documentation](/nomad/docs/integrations/vault-integration#enterprise-configuration) for more information.
 
+## Multiple Vault and Consul Clusters
+
+Nomad administrators can now define multiple integrations in the [vault](/nomad/docs/configuration/vault) and
```

Comment:
"can now" is language from a release note, whereas this is a permanent list of features. It should use the right tense/wording for that.

Reply:
Thank  you.